### ISSUE TITLE
[Bug] Drop requirement for pickle5 in hydra_ray_launcher for python >= 3.8

### ISSUE THREAD

**Issue by colobas** [facebookresearch/hydra#2336]
# 🐛 Bug
## Description
I believe in the same spirit as what was done in this [commit](https://github.com/facebookresearch/hydra/commit/0da8ef59c3331d9c54d1d3b4e9aa51029030ef1c), this [line](https://github.com/facebookresearch/hydra/blob/782eb04eb5ac22e90aad0a440d871eb1bad198b4/plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/_config.py#L182) also needs to be adapted.

## Checklist
- [ ] I checked on the latest version of Hydra
- [ ] I created a minimal repro (See [this](https://stackoverflow.com/help/minimal-reproducible-example) for tips).

## To reproduce

** Minimal Code/Config snippet to reproduce **

** Stack trace/error message **
```
// Paste the bad output here!
```

## System information
- **Hydra Version** :  
- **Python version** : 
- **Virtual environment type and version** : 
- **Operating system** :

**Comment by Jasha10** [comment]
Closed by #2337.

**Comment by thomashirtz** [comment]
I tried today to install `hydra-ray-launcher`, but still got the issue
```
(venv) D:\Thomas\Python\manufacturing>pip install hydra-ray-launcher --upgrade
Collecting hydra-ray-launcher
  Using cached hydra_ray_launcher-1.2.0-py3-none-any.whl (14 kB)
Collecting pickle5==0.0.11
  Using cached pickle5-0.0.11.tar.gz (132 kB)
  Preparing metadata (setup.py) ... done
Collecting cloudpickle==2.0.0
  Using cached cloudpickle-2.0.0-py3-none-any.whl (25 kB)
Collecting ray[default]==1.12.0
  Using cached ray-1.12.0-cp38-cp38-win_amd64.whl (19.9 MB)
Collecting aiohttp==3.8.1
  Using cached aiohttp-3.8.1-cp38-cp38-win_amd64.whl (555 kB)
Collecting boto3==1.22.6
  Using cached boto3-1.22.6-py3-none-any.whl (132 kB)
Requirement already satisfied: hydra-core>=1.1.2 in d:\thomas\python\manufacturing\venv\lib\site-packages (from hydra-ray-launcher) (1.3.1)
Collecting frozenlist>=1.1.1
  Using cached frozenlist-1.3.3-cp38-cp38-win_amd64.whl (34 kB)
Collecting async-timeout<5.0,>=4.0.0a3
  Using cached async_timeout-4.0.2-py3-none-any.whl (5.8 kB)
Collecting aiosignal>=1.1.2
  Using cached aiosignal-1.3.1-py3-none-any.whl (7.6 kB)
Requirement already satisfied: attrs>=17.3.0 in d:\thomas\python\manufacturing\venv\lib\site-packages (from aiohttp==3.8.1->hydra-ray-launcher) (22
.2.0)
Requirement already satisfied: charset-normalizer<3.0,>=2.0 in d:\thomas\python\manufacturing\venv\lib\site-packages (from aiohttp==3.8.1->hydra-ra
y-launcher) (2.1.1)
Collecting yarl<2.0,>=1.0
  Using cached yarl-1.8.2-cp38-cp38-win_amd64.whl (56 kB)
Collecting multidict<7.0,>=4.5
  Using cached multidict-6.0.4-cp38-cp38-win_amd64.whl (28 kB)
Collecting s3transfer<0.6.0,>=0.5.0
  Using cached s3transfer-0.5.2-py3-none-any.whl (79 kB)
Collecting jmespath<2.0.0,>=0.7.1
  Using cached jmespath-1.0.1-py3-none-any.whl (20 kB)
Collecting botocore<1.26.0,>=1.25.6
  Using cached botocore-1.25.13-py3-none-any.whl (8.7 MB)
Requirement already satisfied: requests in d:\thomas\python\manufacturing\venv\lib\site-packages (from ray[default]==1.12.0->hydra-ray-launcher) (2
.28.1)
Collecting jsonschema
  Using cached jsonschema-4.17.3-py3-none-any.whl (90 kB)
Collecting msgpack<2.0.0,>=1.0.0
  Using cached msgpack-1.0.4-cp38-cp38-win_amd64.whl (62 kB)
Requirement already satisfied: protobuf>=3.15.3 in d:\thomas\python\manufacturing\venv\lib\site-packages (from ray[default]==1.12.0->hydra-ray-laun
cher) (3.20.3)
Collecting grpcio<=1.43.0,>=1.28.1
  Using cached grpcio-1.43.0-cp38-cp38-win_amd64.whl (3.4 MB)
Collecting filelock
  Using cached filelock-3.9.0-py3-none-any.whl (9.7 kB)
Requirement already satisfied: click>=7.0 in d:\thomas\python\manufacturing\venv\lib\site-packages (from ray[default]==1.12.0->hydra-ray-launcher)
(8.1.3)
Requirement already satisfied: pyyaml in d:\thomas\python\manufacturing\venv\lib\site-packages (from ray[default]==1.12.0->hydra-ray-launcher) (6.0
)
Requirement already satisfied: numpy>=1.16 in d:\thomas\python\manufacturing\venv\lib\site-packages (from ray[default]==1.12.0->hydra-ray-launcher)
 (1.24.0)
Collecting virtualenv
  Using cached virtualenv-20.17.1-py3-none-any.whl (8.8 MB)
Collecting colorful
  Using cached colorful-0.5.5-py2.py3-none-any.whl (201 kB)
Collecting gpustat>=1.0.0b1
  Using cached gpustat-1.0.0-py3-none-any.whl
Requirement already satisfied: py-spy>=0.2.0 in d:\thomas\python\manufacturing\venv\lib\site-packages (from ray[default]==1.12.0->hydra-ray-launche
r) (0.3.14)
Collecting aiohttp-cors
  Using cached aiohttp_cors-0.7.0-py3-none-any.whl (27 kB)
Collecting prometheus-client<0.14.0,>=0.7.1
  Using cached prometheus_client-0.13.1-py3-none-any.whl (57 kB)
Collecting smart-open
  Using cached smart_open-6.3.0-py3-none-any.whl (56 kB)
Collecting opencensus
  Using cached opencensus-0.11.0-py2.py3-none-any.whl (128 kB)
Requirement already satisfied: antlr4-python3-runtime==4.9.* in d:\thomas\python\manufacturing\venv\lib\site-packages (from hydra-core>=1.1.2->hydr
a-ray-launcher) (4.9.3)
Requirement already satisfied: packaging in d:\thomas\python\manufacturing\venv\lib\site-packages (from hydra-core>=1.1.2->hydra-ray-launcher) (22.
0)
Requirement already satisfied: importlib-resources in d:\thomas\python\manufacturing\venv\lib\site-packages (from hydra-core>=1.1.2->hydra-ray-laun
cher) (5.10.1)
Requirement already satisfied: omegaconf<2.4,>=2.2 in d:\thomas\python\manufacturing\venv\lib\site-packages (from hydra-core>=1.1.2->hydra-ray-laun
cher) (2.3.0)
Requirement already satisfied: urllib3<1.27,>=1.25.4 in d:\thomas\python\manufacturing\venv\lib\site-packages (from botocore<1.26.0,>=1.25.6->boto3
==1.22.6->hydra-ray-launcher) (1.26.13)
Requirement already satisfied: python-dateutil<3.0.0,>=2.1 in d:\thomas\python\manufacturing\venv\lib\site-packages (from botocore<1.26.0,>=1.25.6-
>boto3==1.22.6->hydra-ray-launcher) (2.8.2)
Requirement already satisfied: colorama in d:\thomas\python\manufacturing\venv\lib\site-packages (from click>=7.0->ray[default]==1.12.0->hydra-ray-
launcher) (0.4.6)
Requirement already satisfied: six>=1.7 in d:\thomas\python\manufacturing\venv\lib\site-packages (from gpustat>=1.0.0b1->ray[default]==1.12.0->hydr
a-ray-launcher) (1.16.0)
Requirement already satisfied: psutil>=5.6.0 in d:\thomas\python\manufacturing\venv\lib\site-packages (from gpustat>=1.0.0b1->ray[default]==1.12.0-
>hydra-ray-launcher) (5.9.4)
Collecting blessed>=1.17.1
  Using cached blessed-1.19.1-py2.py3-none-any.whl (58 kB)
Collecting nvidia-ml-py<=11.495.46,>=11.450.129
  Using cached nvidia_ml_py-11.495.46-py3-none-any.whl (25 kB)
Requirement already satisfied: idna>=2.0 in d:\thomas\python\manufacturing\venv\lib\site-packages (from yarl<2.0,>=1.0->aiohttp==3.8.1->hydra-ray-l
auncher) (3.4)
Requirement already satisfied: zipp>=3.1.0 in d:\thomas\python\manufacturing\venv\lib\site-packages (from importlib-resources->hydra-core>=1.1.2->h
ydra-ray-launcher) (3.11.0)
Collecting pkgutil-resolve-name>=1.3.10
  Using cached pkgutil_resolve_name-1.3.10-py3-none-any.whl (4.7 kB)
Collecting pyrsistent!=0.17.0,!=0.17.1,!=0.17.2,>=0.14.0
  Using cached pyrsistent-0.19.3-cp38-cp38-win_amd64.whl (62 kB)
Collecting google-api-core<3.0.0,>=1.0.0
  Using cached google_api_core-2.11.0-py3-none-any.whl (120 kB)
Collecting opencensus-context>=0.1.3
  Using cached opencensus_context-0.1.3-py2.py3-none-any.whl (5.1 kB)
Requirement already satisfied: certifi>=2017.4.17 in d:\thomas\python\manufacturing\venv\lib\site-packages (from requests->ray[default]==1.12.0->hy
dra-ray-launcher) (2022.12.7)
Collecting distlib<1,>=0.3.6
  Using cached distlib-0.3.6-py2.py3-none-any.whl (468 kB)
Collecting platformdirs<3,>=2.4
  Using cached platformdirs-2.6.2-py3-none-any.whl (14 kB)
Requirement already satisfied: wcwidth>=0.1.4 in d:\thomas\python\manufacturing\venv\lib\site-packages (from blessed>=1.17.1->gpustat>=1.0.0b1->ray
[default]==1.12.0->hydra-ray-launcher) (0.2.5)
Collecting jinxed>=1.1.0
  Using cached jinxed-1.2.0-py2.py3-none-any.whl (33 kB)
Collecting googleapis-common-protos<2.0dev,>=1.56.2
  Using cached googleapis_common_protos-1.57.0-py2.py3-none-any.whl (217 kB)
Requirement already satisfied: google-auth<3.0dev,>=2.14.1 in d:\thomas\python\manufacturing\venv\lib\site-packages (from google-api-core<3.0.0,>=1
.0.0->opencensus->ray[default]==1.12.0->hydra-ray-launcher) (2.15.0)
Requirement already satisfied: cachetools<6.0,>=2.0.0 in d:\thomas\python\manufacturing\venv\lib\site-packages (from google-auth<3.0dev,>=2.14.1->g
oogle-api-core<3.0.0,>=1.0.0->opencensus->ray[default]==1.12.0->hydra-ray-launcher) (5.2.0)
Requirement already satisfied: rsa<5,>=3.1.4 in d:\thomas\python\manufacturing\venv\lib\site-packages (from google-auth<3.0dev,>=2.14.1->google-api
-core<3.0.0,>=1.0.0->opencensus->ray[default]==1.12.0->hydra-ray-launcher) (4.9)
Requirement already satisfied: pyasn1-modules>=0.2.1 in d:\thomas\python\manufacturing\venv\lib\site-packages (from google-auth<3.0dev,>=2.14.1->go
ogle-api-core<3.0.0,>=1.0.0->opencensus->ray[default]==1.12.0->hydra-ray-launcher) (0.2.8)
Collecting ansicon
  Using cached ansicon-1.89.0-py2.py3-none-any.whl (63 kB)
Requirement already satisfied: pyasn1<0.5.0,>=0.4.6 in d:\thomas\python\manufacturing\venv\lib\site-packages (from pyasn1-modules>=0.2.1->google-au
th<3.0dev,>=2.14.1->google-api-core<3.0.0,>=1.0.0->opencensus->ray[default]==1.12.0->hydra-ray-launcher) (0.4.8)
Building wheels for collected packages: pickle5
  Building wheel for pickle5 (setup.py) ... error
  error: subprocess-exited-with-error

  × python setup.py bdist_wheel did not run successfully.
  │ exit code: 1
  ╰─> [38 lines of output]
      running bdist_wheel
      running build
      running build_py
      creating build
      creating build\lib.win-amd64-cpython-38
      creating build\lib.win-amd64-cpython-38\pickle5
      copying pickle5\pickle.py -> build\lib.win-amd64-cpython-38\pickle5
      copying pickle5\pickletools.py -> build\lib.win-amd64-cpython-38\pickle5
      copying pickle5\__init__.py -> build\lib.win-amd64-cpython-38\pickle5
      creating build\lib.win-amd64-cpython-38\pickle5\test
      copying pickle5\test\pickletester.py -> build\lib.win-amd64-cpython-38\pickle5\test
      copying pickle5\test\test_pickle.py -> build\lib.win-amd64-cpython-38\pickle5\test
      copying pickle5\test\test_picklebuffer.py -> build\lib.win-amd64-cpython-38\pickle5\test
      copying pickle5\test\__init__.py -> build\lib.win-amd64-cpython-38\pickle5\test
      running build_ext
      building 'pickle5._pickle' extension
      creating build\temp.win-amd64-cpython-38
      creating build\temp.win-amd64-cpython-38\Release
      creating build\temp.win-amd64-cpython-38\Release\pickle5
      "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29333\bin\HostX86\x64\cl.exe" /c /nologo /O2 /W3 /GL /DNDE
BUG /MD -Id:\thomas\python\manufacturing\venv\include -IC:\ProgramData\Anaconda3\include -IC:\ProgramData\Anaconda3\Include "-IC:\Program Files (x8
6)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29333\include" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\ucrt" "
-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\shared" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\um" "-IC:\Prog
ram Files (x86)\Windows Kits\10\include\10.0.18362.0\winrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\cppwinrt" /Tcpickle5/_pi
ckle.c /Fobuild\temp.win-amd64-cpython-38\Release\pickle5/_pickle.obj
      _pickle.c
      "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29333\bin\HostX86\x64\cl.exe" /c /nologo /O2 /W3 /GL /DNDE
BUG /MD -Id:\thomas\python\manufacturing\venv\include -IC:\ProgramData\Anaconda3\include -IC:\ProgramData\Anaconda3\Include "-IC:\Program Files (x8
6)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29333\include" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\ucrt" "
-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\shared" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\um" "-IC:\Prog
ram Files (x86)\Windows Kits\10\include\10.0.18362.0\winrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\cppwinrt" /Tcpickle5/pic
klebufobject.c /Fobuild\temp.win-amd64-cpython-38\Release\pickle5/picklebufobject.obj
      picklebufobject.c
      pickle5/picklebufobject.c(20): warning C4273: 'PyPickleBuffer_FromObject'ÿ: liaison DLL incoh‚rente
      C:\ProgramData\Anaconda3\include\picklebufobject.h(18): note: voir la d‚finition pr‚c‚dente de 'PyPickleBuffer_FromObject'
      pickle5/picklebufobject.c(39): warning C4273: 'PyPickleBuffer_GetBuffer'ÿ: liaison DLL incoh‚rente
      C:\ProgramData\Anaconda3\include\picklebufobject.h(22): note: voir la d‚finition pr‚c‚dente de 'PyPickleBuffer_GetBuffer'
      pickle5/picklebufobject.c(58): warning C4273: 'PyPickleBuffer_Release'ÿ: liaison DLL incoh‚rente
      C:\ProgramData\Anaconda3\include\picklebufobject.h(24): note: voir la d‚finition pr‚c‚dente de 'PyPickleBuffer_Release'
      pickle5/picklebufobject.c(208): warning C4273: 'PyPickleBuffer_Type'ÿ: liaison DLL incoh‚rente
      C:\ProgramData\Anaconda3\include\picklebufobject.h(13): note: voir la d‚finition pr‚c‚dente de 'PyPickleBuffer_Type'
      "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29333\bin\HostX86\x64\link.exe" /nologo /INCREMENTAL:NO /L
TCG /DLL /MANIFEST:EMBED,ID=2 /MANIFESTUAC:NO /LIBPATH:d:\thomas\python\manufacturing\venv\libs /LIBPATH:C:\ProgramData\Anaconda3\libs /LIBPATH:C:\
ProgramData\Anaconda3 /LIBPATH:d:\thomas\python\manufacturing\venv\PCbuild\amd64 "/LIBPATH:C:\Program Files (x86)\Microsoft Visual Studio\2019\Comm
unity\VC\Tools\MSVC\14.28.29333\lib\x64" "/LIBPATH:C:\Program Files (x86)\Windows Kits\10\lib\10.0.18362.0\ucrt\x64" "/LIBPATH:C:\Program Files (x8
6)\Windows Kits\10\lib\10.0.18362.0\um\x64" /EXPORT:PyInit__pickle build\temp.win-amd64-cpython-38\Release\pickle5/_pickle.obj build\temp.win-amd64
-cpython-38\Release\pickle5/picklebufobject.obj /OUT:build\lib.win-amd64-cpython-38\pickle5\_pickle.cp38-win_amd64.pyd /IMPLIB:build\temp.win-amd64
-cpython-38\Release\pickle5\_pickle.cp38-win_amd64.lib
         Cr‚ation de la bibliothŠque build\temp.win-amd64-cpython-38\Release\pickle5\_pickle.cp38-win_amd64.lib et de l'objet build\temp.win-amd64-
cpython-38\Release\pickle5\_pickle.cp38-win_amd64.exp
      G‚n‚ration de code en cours
      Fin de la g‚n‚ration du code
      picklebufobject.obj : error LNK2005: PyPickleBuffer_GetBuffer d‚j… d‚fini(e) dans python38.lib(python38.dll)
      build\lib.win-amd64-cpython-38\pickle5\_pickle.cp38-win_amd64.pyd : fatal error LNK1169: un ou plusieurs symboles d‚finis … diff‚rentes repri
ses ont ‚t‚ rencontr‚s
      error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Community\\VC\\Tools\\MSVC\\14.28.29333\\bin\\HostX86\\x64\\link.exe'
 failed with exit code 1169
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for pickle5
  Running setup.py clean for pickle5
Failed to build pickle5
Installing collected packages: pickle5, opencensus-context, nvidia-ml-py, msgpack, distlib, ansicon, smart-open, pyrsistent, prometheus-client, pla
tformdirs, pkgutil-resolve-name, multidict, jmespath, jinxed, grpcio, googleapis-common-protos, frozenlist, filelock, colorful, cloudpickle, async-
timeout, yarl, virtualenv, jsonschema, botocore, blessed, aiosignal, s3transfer, ray, gpustat, google-api-core, aiohttp, opencensus, boto3, aiohttp
-cors, hydra-ray-launcher
  Running setup.py install for pickle5 ... error
  error: subprocess-exited-with-error

  × Running setup.py install for pickle5 did not run successfully.
  │ exit code: 1
  ╰─> [40 lines of output]
      running install
      d:\thomas\python\manufacturing\venv\lib\site-packages\setuptools\command\install.py:34: SetuptoolsDeprecationWarning: setup.py install is dep
recated. Use build and pip and other standards-based tools.
        warnings.warn(
      running build
      running build_py
      creating build
      creating build\lib.win-amd64-cpython-38
      creating build\lib.win-amd64-cpython-38\pickle5
      copying pickle5\pickle.py -> build\lib.win-amd64-cpython-38\pickle5
      copying pickle5\pickletools.py -> build\lib.win-amd64-cpython-38\pickle5
      copying pickle5\__init__.py -> build\lib.win-amd64-cpython-38\pickle5
      creating build\lib.win-amd64-cpython-38\pickle5\test
      copying pickle5\test\pickletester.py -> build\lib.win-amd64-cpython-38\pickle5\test
      copying pickle5\test\test_pickle.py -> build\lib.win-amd64-cpython-38\pickle5\test
      copying pickle5\test\test_picklebuffer.py -> build\lib.win-amd64-cpython-38\pickle5\test
      copying pickle5\test\__init__.py -> build\lib.win-amd64-cpython-38\pickle5\test
      running build_ext
      building 'pickle5._pickle' extension
      creating build\temp.win-amd64-cpython-38
      creating build\temp.win-amd64-cpython-38\Release
      creating build\temp.win-amd64-cpython-38\Release\pickle5
      "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29333\bin\HostX86\x64\cl.exe" /c /nologo /O2 /W3 /GL /DNDE
BUG /MD -Id:\thomas\python\manufacturing\venv\include -IC:\ProgramData\Anaconda3\include -IC:\ProgramData\Anaconda3\Include "-IC:\Program Files (x8
6)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29333\include" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\ucrt" "
-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\shared" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\um" "-IC:\Prog
ram Files (x86)\Windows Kits\10\include\10.0.18362.0\winrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\cppwinrt" /Tcpickle5/_pi
ckle.c /Fobuild\temp.win-amd64-cpython-38\Release\pickle5/_pickle.obj
      _pickle.c
      "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29333\bin\HostX86\x64\cl.exe" /c /nologo /O2 /W3 /GL /DNDE
BUG /MD -Id:\thomas\python\manufacturing\venv\include -IC:\ProgramData\Anaconda3\include -IC:\ProgramData\Anaconda3\Include "-IC:\Program Files (x8
6)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29333\include" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\ucrt" "
-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\shared" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\um" "-IC:\Prog
ram Files (x86)\Windows Kits\10\include\10.0.18362.0\winrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\cppwinrt" /Tcpickle5/pic
klebufobject.c /Fobuild\temp.win-amd64-cpython-38\Release\pickle5/picklebufobject.obj
      picklebufobject.c
      pickle5/picklebufobject.c(20): warning C4273: 'PyPickleBuffer_FromObject'ÿ: liaison DLL incoh‚rente
      C:\ProgramData\Anaconda3\include\picklebufobject.h(18): note: voir la d‚finition pr‚c‚dente de 'PyPickleBuffer_FromObject'
      pickle5/picklebufobject.c(39): warning C4273: 'PyPickleBuffer_GetBuffer'ÿ: liaison DLL incoh‚rente
      C:\ProgramData\Anaconda3\include\picklebufobject.h(22): note: voir la d‚finition pr‚c‚dente de 'PyPickleBuffer_GetBuffer'
      pickle5/picklebufobject.c(58): warning C4273: 'PyPickleBuffer_Release'ÿ: liaison DLL incoh‚rente
      C:\ProgramData\Anaconda3\include\picklebufobject.h(24): note: voir la d‚finition pr‚c‚dente de 'PyPickleBuffer_Release'
      pickle5/picklebufobject.c(208): warning C4273: 'PyPickleBuffer_Type'ÿ: liaison DLL incoh‚rente
      C:\ProgramData\Anaconda3\include\picklebufobject.h(13): note: voir la d‚finition pr‚c‚dente de 'PyPickleBuffer_Type'
      "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29333\bin\HostX86\x64\link.exe" /nologo /INCREMENTAL:NO /L
TCG /DLL /MANIFEST:EMBED,ID=2 /MANIFESTUAC:NO /LIBPATH:d:\thomas\python\manufacturing\venv\libs /LIBPATH:C:\ProgramData\Anaconda3\libs /LIBPATH:C:\
ProgramData\Anaconda3 /LIBPATH:d:\thomas\python\manufacturing\venv\PCbuild\amd64 "/LIBPATH:C:\Program Files (x86)\Microsoft Visual Studio\2019\Comm
unity\VC\Tools\MSVC\14.28.29333\lib\x64" "/LIBPATH:C:\Program Files (x86)\Windows Kits\10\lib\10.0.18362.0\ucrt\x64" "/LIBPATH:C:\Program Files (x8
6)\Windows Kits\10\lib\10.0.18362.0\um\x64" /EXPORT:PyInit__pickle build\temp.win-amd64-cpython-38\Release\pickle5/_pickle.obj build\temp.win-amd64
-cpython-38\Release\pickle5/picklebufobject.obj /OUT:build\lib.win-amd64-cpython-38\pickle5\_pickle.cp38-win_amd64.pyd /IMPLIB:build\temp.win-amd64
-cpython-38\Release\pickle5\_pickle.cp38-win_amd64.lib
         Cr‚ation de la bibliothŠque build\temp.win-amd64-cpython-38\Release\pickle5\_pickle.cp38-win_amd64.lib et de l'objet build\temp.win-amd64-
cpython-38\Release\pickle5\_pickle.cp38-win_amd64.exp
      G‚n‚ration de code en cours
      Fin de la g‚n‚ration du code
      picklebufobject.obj : error LNK2005: PyPickleBuffer_GetBuffer d‚j… d‚fini(e) dans python38.lib(python38.dll)
      build\lib.win-amd64-cpython-38\pickle5\_pickle.cp38-win_amd64.pyd : fatal error LNK1169: un ou plusieurs symboles d‚finis … diff‚rentes repri
ses ont ‚t‚ rencontr‚s
      error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Community\\VC\\Tools\\MSVC\\14.28.29333\\bin\\HostX86\\x64\\link.exe'
 failed with exit code 1169
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: legacy-install-failure

× Encountered error while trying to install package.
╰─> pickle5

note: This is an issue with the package mentioned above, not pip.
hint: See above for output from the failure.
```

hydra-core 1.3.1
hydra-optuna-sweeper 1.2.0
python 3.8.8
